designs/cloudWatch/cloudWatchIntegration.md

```diff
@@ -0,0 +1,76 @@
+# CloudWatch Logs Experience
+
+* **Type**: UX Design
+* **Status**: Proposal
+
+## Introduction
+CloudWatch Logs enables a user to centralize the logs from all of their systems, applications, and AWS services that they use, in a single, highly scalable service. This document describes adding CloudWatch Logs into the JetBrains toolkit in order to facilitate easily viewing and searching them for specific error codes or patterns.
+
+## Comparison to other products
+
+| Product                       | Difference    |
+|-------------------------------|---------------|
+| AWS Toolkit for Visual Studio | Not supported |
```

Comment:
The Visual Studio Toolkit allows you to pull down a Lambda Function's Logs. Unsure if logs functionality is elsewhere in the toolkit. @normj ?

![image](https://user-images.githubusercontent.com/39839589/74892075-79cfe680-533d-11ea-815b-2f277d8b2f3c.png)

Reply:
Is it logs from cloudwatch or just the tail of the logs from a call to `invokeFunction` ?

Reply:
It is the log stream downloaded to a temp file and opened in Notepad

Reply:
how do they determine which log stream it is in CW for a given invocation?

Reply:
Doesn't seem to, its just a list of all and you select which to download

Reply:
ahhh - k - yeah that's a shame.